### PR TITLE
Update README base on the v0.1-test.2 tag

### PR DESCRIPTION
**Pull request by AstroProfundis**
Update readme file to follow up codebase of v0.1-test.2 tag.

 - Add usage & argument descriptions of `insight.py`
 - Add basic examples
 - Add note of under-development feature of intergration with ansible playbooks

@CaitinChen PTAL

### CODE REVIEW

**Comment by queenypingcap** [comment]
@CaitinChen Please take a look. Thanks!

File: README.md [hunk 1]
```diff
@@ -25,6 +25,73 @@ The following features have been implemented:
...
+If calling `insight.py` with no argument provided, following information are collected and saved to `data` directory of current working directory:
+
+ - `collector`, providing system information, program versions, NTP status, partition statics and process statics of TiDB modules (if present), placed under `collector` sub-directory.
+ - PD API output of `config`, `diagnose`, `health`, `hotspot`, `labels`, `members`, `operators`, `regions`, `regionstats`, `schedulers` and `status`, placed under `pdctl` sub-directory in JSON format.
+
+Using `-o` can set another location to store output data other than the default `data`.
+
+### Examples
+
+A common combination of params that could be useful is: `insight.py -l --config-file`, with which the basic system information, TiDB modules' log files and several config files are collected.
```
Comment (CaitinChen) [inline README.md]: -> Take `insight.py -l --config-file`, a common combination of arguments as an example. It is used to collect the basic system information, TiDB modules' log files and several configuration files.

File: README.md [hunk 2]
```diff
@@ -25,6 +25,73 @@ The following features have been implemented:
...
+
+If you want to collect runtime information of a TiDB cluster from PD on a remote host (instead of running the scripts on that server), set `--pd-host` to the IP address or a resolvable hostname of that host. Additionally, if PD is listening on non-default port, use `--pd-port` to set it.
+
+System log is not collectd by default, even if `-l` is set. If needed, use `-l --syslog` to enable it. Both `syslog` and `systemd-journald` log are supported and automatically detected, note that there is no content filter available for log files, thus enabling log file collecting may leads to very large output size.
```
Comment (CaitinChen) [inline README.md]: -> The system log is not collected by default, even if `-l` is set. If needed, use `-l --syslog` to enable it. Both `syslog` and `systemd-journald` logs are supported and automatically detected. Note that there is no content filter available for log files, thus enabling log file collecting may lead to a very large output size.

File: README.md [hunk 3]
```diff
@@ -25,6 +25,73 @@ The following features have been implemented:
...
+
+## Intergration with Ansible
+
+The tidb-insight project is designed to intergrate with [tidb-ansible](https://github.com/pingcap/tidb-ansible) playbooks.
```
Comment (CaitinChen) [inline README.md]: -> The `tidb-insight` project is designed to integrate with [tidb-ansible](https://github.com/pingcap/tidb-ansible) playbooks.

File: README.md [hunk 4]
```diff
@@ -25,6 +25,73 @@ The following features have been implemented:
...
+
+> All collecting features will be disabled by default after the development of intergration is finished, so it's highly recommended to specify all the args needed when using tidb-insight now, even if the feature is currently enabled by default.
```
Comment (CaitinChen) [inline README.md]: ->
> **Note:** All collecting features will be disabled by default after the development of integration is finished, so it's highly recommended to specify all the arguments needed when using `tidb-insight` now, even if the feature is currently enabled by default.

File: README.md [hunk 5]
```diff
@@ -25,6 +25,73 @@ The following features have been implemented:
...
+
+TO DO: Add documents / links to documents of using tidb-insight with tidb-ansible.
```
Comment (CaitinChen) [inline README.md]: -> TO DO: Add documents/links to documents of using `tidb-insight` with `tidb-ansible`.

File: README.md [hunk 6]
```diff
@@ -25,6 +25,73 @@ The following features have been implemented:
...
+
+    usage: insight.py [-h] [-o OUTPUT] [-p] [--pid PID] [--tidb-proc]
+                    [--perf-exec PERF_EXEC] [--perf-freq PERF_FREQ]
+                    [--perf-time PERF_TIME] [-l] [--syslog] [--config-file]
+                    [--pd-host PD_HOST] [--pd-port PD_PORT]
+
+    TiDB Insight Scripts
+
+    optional arguments:
+    -h, --help            show this help message and exit
```
Comment (CaitinChen) [inline README.md]: **S**how this help message and exit

Comment (AstroProfundis) [reply README.md]: This line is generated by the `argparse` Python module by default.

File: README.md [hunk 7]
```diff
@@ -26,8 +26,8 @@ The following features have been implemented:
  - Record `perf` data of the whole system, TiDB modules, or specific process. (disabled by default)
  - Check the file size of TiDB modules' data directories. (enabled by default)
  - Collect and save log files of TiDB modules or system. (disabled by default)
- - Collect ans save various config files of system. (disabled by default)
- - Query PD API to collect runtime information of TiDB cluster. (enabled by default)
+ - Collect ans save various configuration files of system. (disabled by default)
```
Comment (CaitinChen) [inline README.md]: Collect **and** save various configuration files of the system. (disabled by default)

File: README.md [hunk 8]
```diff
@@ -44,15 +44,14 @@ TiDB Insight Scripts
 optional arguments:
   -h, --help            show this help message and exit
   -o OUTPUT, --output OUTPUT
-                        The dir to store output data of TiDB Insight, any
-                        existing file will be overwritten without futher
+                        The directory to store output data of TiDB Insight,
+                        any existing file will be overwritten without futher
```
Comment (CaitinChen) [inline README.md]: The directory to store the output data of TiDB Insight. **Any** existing file will be overwritten without further confirmation.

File: README.md [hunk 9]
```diff
@@ -44,15 +44,14 @@ TiDB Insight Scripts
...
                         confirmation.
-  -p, --perf            Collect trace info with perf. Default is disabled.
-  --pid PID             PID of process to run perf on, if '-p/--perf' is not
-                        set, this value will be ignored and would not take any
-                        effection. Multiple PIDs can be set by using more than
-                        one --pid args. Default is None and means the whole
-                        system.
+  -p, --perf            Collect trace info using perf. Disabled by default.
+  --pid PID             PID of process to run perf on, if `-p/--perf` is not
```
Comment (CaitinChen) [inline README.md]: PID of process to run perf on. **If** -p/--perf is not set, this value will not take effect. Multiple PIDs can be set by using more than one --pid argument. `None` by default which means the whole system.

File: README.md [hunk 10]
```diff
@@ -61,41 +60,41 @@ optional arguments:
...
 
 Note that some arguments would decrease system performance.
 ```
 
-If calling `insight.py` with no argument provided, following information are collected and saved to `data` directory of current working directory:
+If `insight.py` is called with no argument provided, the following information is collected and saved to `data` directory in the current working directory:
 
- - `collector`, providing system information, program versions, NTP status, partition statics and process statics of TiDB modules (if present), placed under `collector` sub-directory.
- - PD API output of `config`, `diagnose`, `health`, `hotspot`, `labels`, `members`, `operators`, `regions`, `regionstats`, `schedulers` and `status`, placed under `pdctl` sub-directory in JSON format.
+ - `collector`, which provids the system information, program versions, NTP status, partition statistics and process statistics of TiDB modules (if present), placed in the `collector` sub-directory.
```
Comment (CaitinChen) [inline README.md]: `collector` which **provides** the system information, program versions, NTP status, partition statistics and process statistics of TiDB modules (if present), placed in the `collector` sub-directory.

File: README.md [hunk 11]
```diff
@@ -61,41 +60,41 @@ optional arguments:
...
+The system log is not collectd by default, even if `-l` is set. If needed, use `-l --syslog` to enable it. Both `syslog` and `systemd-journald` logs are supported and automatically detected. Note that there is no content filter available for log files, thus enabling log file collecting may lead to a very large output size.
 
 ## Intergration with Ansible
 
-The tidb-insight project is designed to intergrate with [tidb-ansible](https://github.com/pingcap/tidb-ansible) playbooks.
+The `tidb-insight` project is designed to intergrate with [tidb-ansible](https://github.com/pingcap/tidb-ansible) playbooks.
 
-> All collecting features will be disabled by default after the development of intergration is finished, so it's highly recommended to specify all the args needed when using tidb-insight now, even if the feature is currently enabled by default.
+> All collecting features will be disabled by default after the development of intergration is finished, so it's highly recommended to specify all the arguments needed when using `tidb-insight` now, even if the feature is currently enabled by default.
```
Comment (CaitinChen) [inline README.md]: > **Note:** All collecting features will be disabled by default after the development of integration is finished, so it's highly recommended to specify all the arguments needed when using `tidb-insight` now, even if the feature is currently enabled by default.

If it talks about something that the user should pay special attention to,  you can use the NOTE format as above. If not, please delete ">" in your text.

**Pull request merged**